friends/src/components/FriendsForm.js

```diff
@@ -0,0 +1,73 @@
+import React, {Component} from 'react'; 
+import { connect } from 'react-redux'; 
+import { createFriends} from '../actions';
+import '../App.css';
+class FriendForm extends Component {
+    state={
+        name: '', 
+        age: '', 
+        email: '', 
+    }; 
+    handleInputChange = event => {
+        this.setState({ [event.target.name]: event.target.value}); 
+    }; 
+
+    handleAddFriend = _ => {
+        const {name, age, email} = this.state; 
```

Comment:
![image](https://user-images.githubusercontent.com/26180499/47564515-203ae180-d8da-11e8-9c6a-e02497bc7c0c.png)

This fixes your refresh on adding friends.